firedrake/output.py

```diff
@@ -7,7 +6,10 @@
 import weakref
 from pyop2.mpi import COMM_WORLD, dup_comm
 from pyop2.datatypes import IntType
-
+from .paraview_reordering import vtk_lagrange_tet_reorder,\
+    vtk_lagrange_hex_reorder, vtk_lagrange_interval_reorder,\
+    vtk_lagrange_triangle_reorder, vtk_lagrange_quad_reorder,\
+    vtk_lagrange_wedge_reorder
```

Comment:
Ubuntu is easy - as noted by you below, pip install vtk works. When I had a mac about 2 years ago, I never had any issues with this.

Reply:
We really don't want to add a compiled ubuntu/homebrew dependency because this will muck up supercomputer installs. However if we can just pip install, this is fine.

Reply:
Alright. I will add the pip install, revise the pull and make jenkins happy with it.